internal/plugin6/grpc_provider.go

```diff
@@ -511,10 +524,10 @@ func (p *GRPCProvider) PlanResourceChange(r providers.PlanResourceChangeRequest)
 	return resp
 }
 
-func (p *GRPCProvider) ApplyResourceChange(r providers.ApplyResourceChangeRequest) (resp providers.ApplyResourceChangeResponse) {
+func (p *GRPCProvider) ApplyResourceChange(ctx context.Context, r providers.ApplyResourceChangeRequest) (resp providers.ApplyResourceChangeResponse) {
 	logger.Trace("GRPCProvider.v6: ApplyResourceChange")
 
-	schema := p.GetProviderSchema()
+	schema := p.GetProviderSchema(ctx)
```

Comment:
Is there a particular reason for which the `ApplyResourceChange` for the `plugin` package is having `context.WithoutCancel` but the package `plugin6` isn't?

Reply:
No good reason... just a typical hazard of maintaining duplicated code. 🙄 I'll fix it before merging. Thanks!